src/javascript/utils/getButtonRenderer.jsx

```diff
@@ -26,7 +28,7 @@ const useLabel = labelProps => {
 
     return {
         label: (labelStyle === 'none') ? null : label,
-        tooltipLabel: showTooltip ? label : null
+        tooltipLabel: showTooltip ? t(tooltipLabel, tooltipParams) : null
```

Comment:
we should default to label if showTooltip is true and tooltipLabel is not available